tests/workflows/test_image_analysis.py

```diff
@@ -0,0 +1,30 @@
+import dask
+import dask.array as da
+import numpy as np
+from dask_image import ndfilters, ndmeasure, ndmorph
+
+
+def test_BBBC039(small_client):
```

Comment:
Suggest adding a link to [those talk slides](https://genevievebuckley.github.io/dask-image-talk-2020/) and/or [the repo](https://github.com/GenevieveBuckley/dask-image-talk-2020). That way if anyone needs to work on this benchmark later, they can look there for context/answers there before quizzing James about it.